test/wanda/checks_customizations_test.exs

```diff
@@ -616,4 +601,39 @@ defmodule Wanda.ChecksCustomizationsTest do
         where: c.group_id == ^group_id
     )
   end
+
+  defp get_scenario_opts(scenario) do
```

Comment:
super nitpick: I personally find the usage of this helper of not much help unless you wrote it hehe. Kind of implicit logic with "hidden" defaults.

Reply:
either that or the Map.get repeated 😄